readme.md

```diff
@@ -121,9 +121,9 @@ jobs:
 | Name          | Description                                                                         | Default                 | Required |
 | :------------ | :---------------------------------------------------------------------------------- | :---------------------- | :------- |
 | `token`       | A `admin:org`, `read:user`, `user:email` scoped [PAT]                               |                         | `true`   |
-| `report-path` | Report CSV file path within the repository                                          | `invitation-report.csv` | `true`   |
-| `base-sha`    | Report base SHA, from https://github.com/ActionsDesk/ghec-invitations-report-action |                         | `true`   |
-| `head-sha`    | Report head SHA, from https://github.com/ActionsDesk/ghec-invitations-report-action |                         | `true`   |
+| `report_path` | Report CSV file path within the repository                                          | `invitation-report.csv` | `true`   |
+| `base_sha`    | Report base SHA, from https://github.com/ActionsDesk/ghec-invitations-report-action |                         | `true`   |
```

Comment:
As you are changing the name of those parameters, just a heads up to add a note on changelog for the breaking change.